avro-data/src/test/avro/ArrayOfRecordsWithDefault.avsc

```diff
@@ -0,0 +1,36 @@
+{
+  "type": "record",
+  "name": "ArrayOfRecordsWithDefault",
+  "namespace": "com.connect.avro",
+  "fields": [
+    {
+      "name": "name",
+      "type": [null, "string"],
```

Comment:
`null` should be in double quotes as `"null"`

Reply:
sorry I missed the quote. I will have it fixed in the next commit